### PR TITLE
Add missing options to member types

### PR DESCRIPTION
**Pull request by mtneug**
For #2055

This adds Backup, MonitorAddress, MonitorPort and Tags to member types.

Links to the line numbers/files in the OpenStack source code that support the code in this PR:

**`CreateMemberOpts`**

[Documentation](https://docs.openstack.org/api-ref/load-balancer/v2/?expanded=create-member-detail#create-member)
[Code](https://github.com/openstack/octavia/blob/198980639cbe02820f90783fa04d5239717ed80d/octavia/api/v2/types/member.py#L87)

**`UpdateMemberOpts`**

[Documentation](https://docs.openstack.org/api-ref/load-balancer/v2/?expanded=update-a-member-detail#update-a-member)
[Code](https://github.com/openstack/octavia/blob/198980639cbe02820f90783fa04d5239717ed80d/octavia/api/v2/types/member.py#L100-L104)

**`BatchUpdateMemberOpts`**

[Documentation](https://docs.openstack.org/api-ref/load-balancer/v2/?expanded=batch-update-members-detail#batch-update-members)
[Code](https://github.com/openstack/octavia/blob/198980639cbe02820f90783fa04d5239717ed80d/octavia/api/v2/types/member.py#L79-L87)

### CODE REVIEW

File: openstack/loadbalancer/v2/pools/requests.go [hunk 1]
```diff
@@ -390,6 +411,21 @@ type BatchUpdateMemberOpts struct {
...
 	// or false (DOWN).
 	AdminStateUp *bool `json:"admin_state_up,omitempty"`
+
+	// Is the member a backup? Backup members only receive traffic when all
+	// non-backup members are down.
+	// Requires microversion 2.1 or later.
+	Backup *bool `json:"backup,omitempty"`
+
+	// An alternate IP address used for health monitoring a backend member.
+	MonitorAddress string `json:"monitor_address,omitempty"`
```
Comment (jtopjian) [inline openstack/loadbalancer/v2/pools/requests.go:421]: Is removing/unsetting a monitor address from a member possible? If so, is it done via an empty string (`""`) or with nil/null?

The combination of `string` and `omitempty` will cause values of empty strings to be removed from the request body. If an empty string is used to remove/unset the monitor address, this should be `*string`.

**Review by jtopjian — COMMENTED** [review]
@alex Thank you for submitting this. I've left one question.

File: openstack/loadbalancer/v2/pools/requests.go (hunk 1 above)
Comment (mtneug) [reply openstack/loadbalancer/v2/pools/requests.go:421]: Yes, it is possible. I tested it and for me

* setting it to `""`
* setting it to `null`
* omitting it

will remove/unset the monitor address. I'm unsure what the preferred way is.

Comment (jtopjian) [reply openstack/loadbalancer/v2/pools/requests.go:421]: I recommend setting it to:

```go
MonitorAddress *string `json:"monitor_address,omitempty"`
```

**Review by jtopjian — APPROVED** [review]
Thank you for working on this. I've noted one small change, but I can fix this post-merge.

edit: I have no idea why Github auto-corrected my at reply to "Alex" and not @mtneug.

**Pull request merged**